### ISSUE TITLE
Guardr is a Drupal distribution need in backdropcms

### ISSUE THREAD

**Issue by domaingood** [backdrop/backdrop-issues#3998]
**Description of the need**
Guardr is a Drupal distribution with a combination of modules and settings to enhance a Drupal application's security and availability to meet enterprise security requirements.

**Proposed solution**
Please add Guardr Drupal distribution

**Alternatives that have been considered**
Describe alternatives you've considered. Have you seen this work in a better way in another CMS?

Is there a contributed module that accomplishes this? ...if so, then has this
been ported to Backdrop?: https://www.drupal.org/project/guardr

**Comment by klonos** [comment]
Hello @domaingood and welcome to Backdrop! ...this is the issue queue for Backdrop core; can you please file a new issue here: https://github.com/backdrop-ops/contrib/issues and mark it as a port request?

I don't think that anyone has ported this yet (you can search for existing ported or new contributed projects here: https://github.com/backdrop-contrib) in fact, I don't think that any distribution for Backdrop exists yet.

You can try searching which modules that are included in Guardr have already been ported. This should give you an idea of how easy it would be to port Guardr to Backdrop.

**Comment by klonos** [comment]
...I have looked, and there seemed to be interest to port modules included in that distro. See: https://github.com/backdrop-ops/contrib/issues/162

**Comment by deekayen** [comment]
I already ported most of the guardr modules to Backdrop - I just didn't port the distribution wrapper. When I ported them, I also didn't necessarily do a direct port, either. Like instead of password policy getting a direct port, I turned it into passphrase policy. Login security became Previous login and Disable login errors. Session Limit, Diskfree, Remove Generator, and Paranoia are mostly the same.

Creating a branded distro has some ongoing maintenance cause it needs to be touched every time there's a core release (or at least a core security update) and I'm not at a Drupal or Backdrop shop right now, so I'm not prepared to make any routine commitments.
